clefos/content.md

```diff
@@ -0,0 +1,92 @@
+# ClefOS
+
+ClefOS Linux is a community-supported distribution for IBM Z (aka "mainframe") derived from sources freely provided to the public by [CentOS](http://vault.clefos.org/) which in turn is derived from the [Red Hat](ftp://ftp.redhat.com/pub/redhat/linux/enterprise/) sources for Red Hat Enterprise Linux (RHEL). As such, ClefOS Linux aims to be functionally compatible with CentOS and RHEL. The ClefOS Project mainly changes packages to remove upstream vendor branding and artwork. ClefOS Linux is no-cost and free to redistribute. Each ClefOS Linux version is maintained and released according to the CentOS schedule.
+
+%%LOGO%%
+
+## ClefOS image documentation
+
+The `clefos:latest` tag is always the most recent version currently available.
+
+### Building the Base Image
+
+The image is built via the `make` command which will create the tarball, build the image, and push the image to the repository.
```

Comment:
This isn't entirely accurate for the official image, since it's not pushed via `docker push` from that repo, so leaving off the last bit of this is probably sufficient.